sitetree/management/commands/sitetree_resync_apps.py

```diff
@@ -63,6 +63,9 @@ def handle(self, *apps, **options):
                             item.save(using=using)
                             # Copy permissions to M2M field once `item`
                             # has been saved
-                            item.access_permissions = item.permissions
+                            if hasattr(item.access_permissions, "set"):
```

Comment:
Here and in other cases please use single quotes `'` for string literals to not to contradict overall style and reduce visual noise.